src/material/slide-toggle/_slide-toggle-theme.scss

```diff
@@ -40,23 +39,30 @@
     handle-surface-color: surface,
     unselected-handle-color: $on-surface-variant,
 
-    selected-icon-color: $selected-icon-color,
+    selected-icon-color: $icon-color,
     disabled-selected-icon-color: $icon-color,
     disabled-unselected-icon-color: $icon-color,
     unselected-icon-color: $icon-color,
   );
 }
 
 // Generates the mapping for the properties that change based on the slide toggle color.
-@function _get-theme-color-map($color-palette) {
-  $state-content: color.scale($color-palette, $blackness: 50%);
-  $inverse: color.scale($color-palette, $lightness: 75%);
+@function _get-theme-color-map($color-palette, $is-dark) {
+  $primary: theming.get-color-from-palette($color-palette, 600);
```

Comment:
nit: I'm not sure that calling it "primary" here is right since it could be the accent or warn color as well.

Reply:
I'm using primary to align with what MDC calls it. Looking at this: https://source.corp.google.com/piper///depot/google3/javascript/materialdesign/tokens/gm/versions/v0_48/_gm-comp-switch.scss;rcl=480228505;l=16